server/app/routers/api/router.js

```diff
@@ -6,9 +6,17 @@ const router = express.Router();
 // Import And Use Routers Here
 /* ************************************************************************* */
 
-const itemsRouter = require("./items/router");
+const usersAppRouter = require("./users/userAppRouter");
```

Comment:
userRouter would be a more appropriate name to respect your naming convention for other routes.